README.md

```diff
@@ -43,9 +43,22 @@ The SDO owner services are packaged as a single docker container that can be run
   chmod +x run-sdo-owner-services.sh
   ```
 
-2. Run `./run-sdo-owner-services.sh -h` to see the usage, and set all of the necessary environment variables.
+2. Run `./run-sdo-owner-services.sh -h` to see the usage, and set all of the necessary environment variables. Export your environment variables `HZN_EXCHANGE_URL`, `HZN_FSS_CSSURL`, `HZN_ORG_ID`, and `HZN_EXCHANGE_USER_AUTH` for the subsequent steps.
```

Comment:
Don't list the env var names here, because they are listed right below in the cmd block. That avoids the possibility of someone later on adding a new env var to one list and forgetting the other.